### ISSUE TITLE
Grpc.Core Exception - Unhanded exception failed to connect to all addresses 

### ISSUE THREAD

**Issue by anilraut30** [zeebe-io/zeebe-client-csharp#59]
**Grpc Exception - Unhanded exception failed to connect to all addresses**

As stated in the documentation, I started in Sample run steps, I started the zeebe client first which seems running with default setting and then Client.Examples cs project which resulting into gRPC exception

**To Reproduce**
1. Start zeebe client
`docker run -p 26500:26500 camunda/zeebe:latest`

2. Git checkout C Sharp client.
3. Cd Client.Examples
4. Run C Sharp Sample project as:
`dotnet run --project Client.Examples.csproj `
5. Error
![image](https://user-images.githubusercontent.com/8069273/69360878-fc6adc00-0c50-11ea-8cd7-0ba71de817b3.png)

I assume Zeebe client in step # 1 is the only prerequisite as stated for this sample run...

Am I missing anything?

**Expected behavior**
As I understand, client uses Zeebe orchestration client, deploy the workflow file and complete the flow as per workflow.

**Enviroment (please complete the following information):**
 - OS: Windows 10

**Comment by Zelldon** [comment]
Hey @anilraut30 thanks for reporting this and to provide these steps. I followed them, but can't reproduce this failure. 

I get the following output:
```
Brokers:

 Address: 172.17.0.2:26501,
 Host: 172.17.0.2,
 NodeId: 0,
 Port: 26501,
 Partitions:
  PartitionId: 1,
  IsLeader: True,
  Role: LEADER
2019-11-21 19:55:01.6076 | DEBUG | Zeebe.Client.Impl.Worker.JobWorker | Job worker (zell-arch) for job type payment-service has been opened. 
2019-11-21 19:55:01.6642 | DEBUG | Zeebe.Client.Impl.Worker.JobWorker | Job worker (zell-arch) activated 5 of 5 successfully. 
Handling job: Key: 2251799813685272, Type: payment-service, WorkflowInstanceKey: 2251799813685252, BpmnProcessId: order-process, WorkflowDefinitionVersion: 1, WorkflowKey: 2251799813685250, ElementId: taskA, ElementInstanceKey: 2251799813685265, Worker: zell-arch, Retries: 3, Deadline: 11/21/19 7:55:11 PM, Variables: {"foo":"123","a":"123","wow":"this"}, CustomHeaders: {}
```

I'm sorry that you have problem with running the example. It might be a windows and docker related problem? I can try to reproduce this tomorrow on windows. I normally run that on linux.

**Comment by Zelldon** [comment]
Could you try to change this [line](https://github.com/zeebe-io/zeebe-client-csharp/blob/master/Client.Examples/Program.cs#L31)

```
        private static readonly string ZeebeUrl = "0.0.0.0:26500";
```

In the example to either 

```
        private static readonly string ZeebeUrl = "localhost:26500";
```

or 


```
        private static readonly string ZeebeUrl = "127.0.0.1:26500";
```

And rerun your procedure?

**Comment by ngprice** [comment]
I was able to reproduce this issue on Docker for Windows, and tried the fix above with "127.0.0.1:26500". This fixed the problem.

**Comment by Zelldon** [comment]
Hey @ngprice thanks for testing that. I will close this issue then 👍
